src/ontology/config/properties.txt

```diff
@@ -15,8 +15,8 @@ http://www.geneontology.org/formats/oboInOwl#source
 http://www.geneontology.org/formats/oboInOwl#hasSynonymType
 http://www.geneontology.org/formats/oboInOwl#SynonymTypeProperty
 http://purl.obolibrary.org/obo/mondo#GENERATED
-http://purl.obolibrary.org/obo/mondo#omim_included
-http://purl.obolibrary.org/obo/mondo#omim_formerly
+http://purl.obolibrary.org/obo/mondo#includedEntryInOMIM
```

Comment:
Spoke with Nico today: [conclusions](https://docs.google.com/document/d/1zsfl5rkxdAhCk9xFX6oBDSa5UAjncBsloFuKU9Yt4Lo/edit?disco=AAABacqdWMQ).

- [ ] @joeflack4 Keep `includedEntryInOMIM`
- [ ] @joeflack4 Also keep `omimIncluded`

Reply:
No, includedEntryInOMIM has nothing to do here; it is not a property. It is a source code value. Yes. It should be kept in mondo. No, it should not be kept in this list.

Reply:
Yes you're right. Not sure why I put that checkbox on this particular PR. But it does also need:
- [ ] @joeflack4 `omimFormerly`